DevOps/frankfurter/chrommunal/bashrc

```diff
@@ -6,3 +6,11 @@ fi
 alias dawn='sudo startxiwi -n trusty /opt/dawn/dawn'
 alias term='sudo enter-chroot'
 alias ubuntu='sudo startunity -X xorg'
+alias screenfetch='/./usr/local/bin/screenfetch-dev -E'
```

Comment:
nvm, it's fine. but i don't think that file is executable, so you got to run it with bash. Also, when i tried to run it on my computer, I had to use sudo. Don't know if this is true for chromebooks